docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md

```diff
@@ -35,18 +41,24 @@ while the other pods shouldn't tolerate the taint. The taint is removed once the
 
 *Warning:* currently there is no guarantee which node is chosen and which pods are being killed
 in order to schedule critical pods, so if rescheduler is enabled your pods might be occasionally
-killed for this purpose.
+killed for this purpose. Please ensure that rescheduler is enabled along with priorities & preemptions in default-scheduler as it may have unwanted side-effects.
```

Comment:
Actually, Rescheduler should not be enabled with "priority and preemption". Rescheduler is oblivious to priority and if enabled, it might preempt high priority pods, instead of low priority ones.

Reply:
Oops. Typo again. I meant should not be enabled. Thanks for catching it.